src/cantools/database/can/formats/sym.py

```diff
@@ -775,14 +775,19 @@ def _get_enum_name(signal: Signal) -> str:
 def _dump_choice(signal: Signal) -> str:
     # Example:
     # Enum=DPF_Actv_Options(0="notActive", 1="active", 2="rgnrtnNddAtmtcllyInttdActvRgnrt", 3="notAvailable")
+    nl = '\n'
+
     if not signal.choices:
         return ''
 
     enum_str = f'Enum={_get_enum_name(signal)}('
-    is_first_choice = True
-    for choice_number, choice_value in signal.choices.items():
-        enum_str += f'{", " if not is_first_choice else ""}{choice_number}="{choice_value}"'
-        is_first_choice = False
+    for choice_count, (choice_number, choice_value) in enumerate(signal.choices.items()):
+        if choice_count % 10 == 0 and not choice_count == 0:
+            enum_str += ","
+            enum_str += nl
```

Comment:
this seems to be too indirect:
```suggestion
            enum_str += ',\n'
```

Reply:
My understanding is that this way would be more universal, as "\" characters can't be added to fStrings in Python versions prior to 3.12. I'm totally onboard with updating, just want to make sure we're aligned.

Reply:
if I'm not horribly mistaken, the proposed code is exactly equivalent (albeit slightly more performant) for any python version featuring f-strings: Prior to python 3.12, it is just not possible to nest strings that are using the same delimiter (a la `f"the song goes: {", ".join(["la"]*10)}"` is not valid prior to python 3.12). that should not be done anyway; instead, use different string delimiters (like `f'the song goes: {", ".join(["la"]*10}'`)

Reply:
Thanks for the clarification. Adding this now.